pkg/utils/print.go

```diff
@@ -0,0 +1,121 @@
+package utils
+
+import (
+	"encoding/json"
+	"fmt"
+	pd "github.com/PagerDuty/go-pagerduty"
+	"github.com/andygrunwald/go-jira"
+	"github.com/openshift-online/ocm-cli/pkg/dump"
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	sl "github.com/openshift/osdctl/internal/servicelog"
+	"github.com/openshift/osdctl/pkg/printer"
+	"math"
+	"os"
+	"strings"
+	"time"
+)
+
+const (
+	delimiter = ">> "
+)
+
+func PrintServiceLogs(serviceLogs []sl.ServiceLogShort, verbose bool, sinceDays int) {
+	var name = fmt.Sprintf("Service Logs in the past %v days", sinceDays)
+	fmt.Println(delimiter + name)
+
+	if verbose {
+		marshalledSLs, err := json.MarshalIndent(serviceLogs, "", "  ")
+		if err != nil {
+			_, _ = fmt.Fprintf(os.Stderr, "Couldn't prepare service logs for printing: %v", err)
+		}
+		_ = dump.Pretty(os.Stdout, marshalledSLs)
+	} else if len(serviceLogs) == 0 {
+		fmt.Println("None")
+	} else {
+		// Non-verbose only prints the summaries
+		for i, errorServiceLog := range serviceLogs {
+			var serviceLogSummary string
+			if errorServiceLog.InternalOnly {
+				internalServiceLogLines := strings.Split(errorServiceLog.Description, "\n")
+				if len(internalServiceLogLines) > 0 {
+					// if the description is "", Split returns []
+					serviceLogSummary = fmt.Sprintf("INT %s", internalServiceLogLines[0])
+				} else {
+					serviceLogSummary = errorServiceLog.Summary
+				}
+			} else {
+				serviceLogSummary = errorServiceLog.Summary
+			}
+			serviceLogSummaryAbbreviated := serviceLogSummary[:int(math.Min(40, float64(len(serviceLogSummary))))]
+			fmt.Printf("%d. %s (%s)\n", i, serviceLogSummaryAbbreviated, errorServiceLog.CreatedAt.Format(time.RFC3339))
+		}
+	}
+}
+
+func PrintPDAlerts(incidents map[string][]pd.Incident, serviceIDs []string) {
+	var name = "PagerDuty Alerts"
+	fmt.Println(delimiter + name)
+
+	if len(serviceIDs) == 0 {
+		fmt.Println("No PD Service Found")
+		return
+	}
+
+	for _, ID := range serviceIDs {
+		fmt.Printf("Service: https://redhat.pagerduty.com/service-directory/%s\n", ID)
```

Comment:
```suggestion
```
I think functions in the `util` package should avoid printing to stdout/err whenever possible